ansible_collections/f5networks/f5_modules/plugins/modules/bigip_policy_rule.py

```diff
@@ -372,9 +373,15 @@
           - This parameter is only valid with the C(ssl_extension) type.
         type: list
         elements: str
+      address_matches_with_any:
+        description:
+          - A list of address strings the tcp should match.
+          - This parameter is only valid with the C(tcp) type.
+        type: list
+        elements: str
       event:
         description:
-          - Events on which conditions such as SSL Extension can be triggered.
+          - Events on which conditions such as SSL Extension/tcp can be triggered.
```

Comment:
Add more information on what is the value for TCP users should be using

Reply:
added event choices